airbyte-integrations/connectors/source-monday/source_monday/pagination_workaround.py

```diff
@@ -0,0 +1,54 @@
+from dataclasses import InitVar, dataclass
+from typing import Any, List, Mapping, Optional, Union
+
+import requests
+from airbyte_cdk.sources.declarative.interpolation import InterpolatedString
+from airbyte_cdk.sources.declarative.requesters.paginators.strategies.pagination_strategy import PaginationStrategy
+from airbyte_cdk.sources.declarative.types import Config
+from dataclasses_jsonschema import JsonSchemaMixin
+
+
+@dataclass
+class PageIncrementWorkaround(PaginationStrategy, JsonSchemaMixin):
```

Comment:
why do we need this workaround?

Reply:
We need `page_size` to be optional and to be calculated on fly for those cases when we don't know exactly how many records we have for some endpoint. We also need `page` to start from 1

Reply:
Removed custom component. Managed it with default pagination strategy